src/components/WebhookService.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { useState } from 'react'
+import { Button, Alert, AlertIcon, AlertTitle } from '@chakra-ui/react'
+
+const WebhookService = () => {
+	const [feedbackMessage, setFeedbackMessage] = useState('')
+	const WebhookURL =
+		'https://discord.com/api/webhooks/1085559987394322464/lrpVJYrvaA_ZBrJCtmVIN_xetza8pQVVZeUb1OHIS8HubCmKItanZQxFyJF-ZhUmer5k'
```

Comment:
Ideally, this should be moved in .env variables, that are git ignored and referenced from there.